src/handlers/register_client.rs

```diff
@@ -67,15 +67,9 @@ pub async fn handler(
         );
     }
 
+    // TODO: ideally we allow for clients to update their device tokens
```

Comment:
I'd rather leave this as it is and then have this (or a different) PR change this to upsert functionality as seen in the notifications store

Reply:
sure